docs/configuration.md

```diff
@@ -456,6 +456,13 @@ Apart from these, the following properties are also available, and may be useful
     from JVM to Python worker for every task.
   </td>
 </tr>
+<tr>
+  <td><code>spark.python.task.killTimeout</code></td>
+  <td>2s</td>
+  <td>
+    How long to wait before killing the python worker if a task cannot be interrupted.
```

Comment:
"Timeout to wait before killing the python worker when a task cannot be interrupted."

Reply:
updated and fix confilct, please have a review, thanks, @zsxwing @jiangxb1987